src/Stratis.Bitcoin.Features.Consensus/PowConsensusValidator.cs

```diff
@@ -22,17 +26,25 @@ public class PowConsensusValidator
         /// <summary>Provider of block header hash checkpoints.</summary>
         protected readonly ICheckpoints Checkpoints;
 
+        /// <summary>Consensus parameters.</summary>
         public NBitcoin.Consensus ConsensusParams { get; }
 
+        /// <summary>Consensus options.</summary>
         public PowConsensusOptions ConsensusOptions { get; }
 
+        /// <summary>Keeps track of how much time different actions took to execute and how many times they were executed.</summary>
         public ConsensusPerformanceCounter PerformanceCounter { get; }
 
         public bool UseConsensusLib { get; set; }
 
         /// <summary>Provider of time functions.</summary>
         protected readonly IDateTimeProvider dateTimeProvider;
 
+        /// <inheritdoc />
```

Comment:
from where?

Reply:
i mean no interface, no base class ... and not override

Reply:
Oh. I thought constructors can inherit class summary.

Reply:
class summary is more of a big picture, constructors do have their own